rolling/release-el-build.yaml

```
@@ -0,0 +1,141 @@
+    KWMv9JbZtx8iHj+wAGGpamRmaNiu1bUJ3B9FHD1im3AZPsIblcZePFtv3QARAQAB
+    tEdST1MgQnVpbGRmYXJtIFNUQUdJTkcgKFJPUyBCdWlsZGZhcm0gU1RBR0lORykg
+    PHN0ZXZlbkBvcGVucm9ib3RpY3Mub3JnPokCVAQTAQgAPhYhBBFJujgLvdWIxaTv
+    kp6hOWOIy3oWBQJfmhUiAhsDBQkB4TOABQsJCAcCBhUKCQgLAgQWAgMBAh4BAheA
+    AAoJEJ6hOWOIy3oWcxYP/2WUOVEDdYyOlnbkwM7plvbnk/Hc38tWs5G3nvUQ5sFn
+    /nJA31XVeEqMKmSbM1p21epMmmMEV9h7cslE730ml1w5zqf5EdnY3KEulFv2liYu
+    N6tNuaVh7nJEn9K1slYOvqjXC0e0PQwhL514UiMAz/dXp+rhZJg39lHBzOJycyug
+    M6zeFF/ybNTs507aumYoEyEGPXC4RExt4Q9iRm/xq31ISnfZPuljzsIWbf+zClcg
+    qFkKCkWGYoV03VbgeVFeD+AevZv5qdU12vSivQrjFRK3u0S0Lfvg2JFdHxmKVwmz
+    WpRgEmwGbEUpS4gQsQcxv5k6ApIICkvsqm78PUZNKKMEyF7xovqvP7mY5DnKlJUv
+    yFRsizgx0SlCse+4eNAQA0O42aWxdvv9KVoEn9g0t38t++7lu5FHWZSxVN6SoNvP
+    WGI2/h0Ws8BsZe/LQipeFF6qMR+0ZmzOT3dUz1izGBDr9QgQuq7athEaZYg4Yqjt
+    +2hhOW3vvlxqhOR18l6M8VBoa31ssNPlctdpPPKHE2xaJUg3MIkMlPS6vGp+nElQ
+    upq0itjzkNAtWGsTNjfR447SJc2BRZFpUcCGy01jFyKjg60kVxTRQAQ2s4vFeUDt
+    E9jppe5QWFAR7rYGgbN/UzJALOFn238V6p1/51RcEDrehl5QEB1pn34ltd381CJ1
+    uQINBF+aFSIBEACtFQV7enAsSDsjsUQc4n1LVif/hjviDKv0k6dmjPhTVyypja9I
+    yWrLpAW1WB5LOMw4ss8KWKKAajY1AoXjHilNsylBNrx9aLhrWNxboLHiagaIiuza
+    woSn1TqWzWSwGzZfM/N+Ok7Ld8e+LJZ+YoBlPzRiSGzo5R3AiNbf0VvT6HyuNsYK
+    x47g28wnvHzU2/efYV8ZYEIpgNF1HkPlIn4ugJXPFyPevRFzOZ2UlXx7URgcG+D/
+    U4sxG4otPUfNBLJjnUSyrhxsV38HIbJ82UKKmvXwkje/OuuHtLmbDvw/2+/e45OQ
+    xAaZvOXM8P3RjVDbd27CFtMMke4hwlqHaHywhIAnW2LvkqRBeX36TXccnN9w0yih
+    GPobhMY8hATbOw49v6JhA2TenSOcNvHdTEob7o08wAnhI/rnNlYXKZZKtR4w7jLx
+    ZWTa2Q6wlgbl69RLnGlPN9i/FNfWlH+faVbIMibzbUOtEBRUZ/qqhAeMwjBwdbby
+    E6GwDypmEYsAmbLeAX0dZnt2SXB3/sggRPKIPLP2PDlR3fj55tBqWsteSRGO8fh+
+    JzYV9T4TxtULoJW85tJGB5odxMAj/z/xMAvT0V4R7PVV4HhKoIXp76LKEOhmkID2
+    JZQ2eXiKrQ40asQ/XjM62Nrk+U7pyK0EvJp3TJRTL3Hv3RTZqfIPstZeuwARAQAB
+    iQI8BBgBCAAmFiEEEUm6OAu91YjFpO+SnqE5Y4jLehYFAl+aFSICGwwFCQHhM4AA
+    CgkQnqE5Y4jLehYVeQ/+LzrmjCoIjuSkCUU4aF/wr/sMY8vAWxdsCfZpswovkT9O
+    K3n/L27Pn50+BUjYIXVWfWuPIMMjkHiYOrmHjN/ewJcJ19jqbxYeROgXxnn8sW+V
+    DNB8GNlPQyhX7izfL5YXYOr6QqIA97iy99DoqxNDS+rLOOLkeuHGXHNWFkh+aKgr
+    FfgPLrkocWlo40lgzUck1H6951LaxJ+BwyOjS83UC7qYUpxSLIXYcFu2WRzKOn9j
+    s5iFrHUUfUrUb7KVSM39RjNwTskI8SRIq4rD0WeAGGvK5wWmn9cIpYfndJ80cRDS
+    7CrGm3DSECBQHCJX4eFolXo7P4jKrPo6o78Of5zjuox114VANK6YaJZarCDlsN4f
+    6PyhS58uZmUVtFwwjMlmNIxqSxxjyAt/b5Ihdp1Qg+eqOxpzmW2w3Nmpsh1G2ih0
+    AhSVv0T7x/nSAmn1wg9qiwaNsxqwvKVnlRuM317K3eHhgYh+vvhYPWwVG76OPbl0
+    ilm1Y8GvBVFPlHogS13mnE9xWvUv0UI/TtMToV7cHbZ04QKjudfy55/US5Zb4tiV
+    CDBbxJOXHfS9UxRg38u5ohVoTVwpJ4+OTlUKZ3KVvOeWCOb12fHcXo5OxJQobQij
+    Hn2+HdjGoRBTY86lZFRm5/L5OEURQ+KFclauvu2554Xs25Us2K8C+VsRcfH8Sp0=
+    =/1E+
+    -----END PGP PUBLIC KEY BLOCK-----
+    urls:
+  - http://repo.ros2.org/el/building/$releasever/$basearch/
+target_repository: http://repo.ros2.org/el/building
+targets:
+  rhel:
+    '7':
+      x86_64:
+type: release-build
+upload_credential_id: 0ddb9953-40d3-4922-80be-298086e30370
```

Should this be a named credential, either `jenkins-agent` or `pulp_admin`?

---

It should be `pulp_admin`. You're right.
```suggestion
upload_credential_id: pulp_admin
```